### PR TITLE
Gagik

### PR DESCRIPTION
**Pull request by StepanyanArman**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,25 @@
...
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-  </body>
-</html>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
```
Comment (albertBarsegyan) [inline public/index.html]: Why you included this CSS file?

File: src/components/Button/LoginBut.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useNavigate } from "react-router-dom";
+import { RouteConstant } from "../../constants/RouteCostant";
+import style from './button.module.css'
+
+
+function Login() {
+    const navigation = useNavigate();
+
+
+    return <button onClick={() => navigation(RouteConstant.LoginPage)}>Sing In <i class="fa-solid fa-user"></i>  </button>
```
Comment (albertBarsegyan) [inline src/components/Button/LoginBut.jsx:10]: you can use <Link>  instead useNavigate hook, useNavigate is for special situations.

File: src/components/Header/header.module.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+:root {
```
Comment (albertBarsegyan) [inline src/components/Header/header.module.css:1]: put variables in varieable.css file, make variable CSS file in styles folder

File: public/index.html (hunk 1 above)
Comment (Spartak-Sargsyan) [reply public/index.html]: This is a link for icons to make FontAwesome work

File: src/components/Header/header.module.css (hunk 3 above)
Comment (Spartak-Sargsyan) [reply src/components/Header/header.module.css:1]: creat new global variable.css file?

**Review by albertBarsegyan — APPROVED** [review]

**Pull request merged**